addons/etcd/opsdefinitions/defragmentation.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: apps.kubeblocks.io/v1alpha1
+kind: OpsDefinition
+metadata:
+  name: etcd-defragmentation
+spec:
+  preConditions:
+    - rule:
+        expression: '{{ eq .component.status.phase "Running" }}'
+        message: "Component is not in Running status."
+  componentInfos:
+  - componentDefinitionName: etcd-v3.5.15
+  actions:
+    - name: etcd-defragmentation
+      failurePolicy: Fail
+      workload:
+        type: Job
+        backoffLimit: 2
+        podSpec:
+          containers:
+            - name: etcd-defragmentation
+              image: quay.io/coreos/etcd:v3.5.6
+              imagePullPolicy: IfNotPresent
+              command:
+                - bash
+                - -c
+                - |
+                  set -e
+                  ENDPOINT=http://${KB_CLUSTER_COMP_NAME}-0.${KB_COMP_HEADLESS_SVC_NAME}.${KB_OPS_NAMESPACE}.svc.cluster.local:2379
```

Comment:
.cluster.local -> ${CLUSTER_DOMAIN}

Reply:
The etcd pod name is fixed to ${KB_CLUSTER_COMP_NAME}-0. However, if instance 0 is offline, the backup process will fail.

Reply:
refactor to use pod env and tlsFile make defragmentation